controller/lib/hal/hal.h

```diff
@@ -206,6 +219,14 @@ class HalApi {
   //
   void test_serialPutIncomingData(const char *data, uint16_t len);
 #endif
+  // Initializes watchdog
+  void watchdog_init();
+
+  // Performs the deviece soft-reset
+  [[noreturn]] void reset_device();
+
+  // Pets the watchdog
```

Comment:
lololol

Reply:
added a description to people who might not be familiar with lingo :)